components/ToolMoodMeasurer/ToolMoodMeasurer.js

```diff
@@ -0,0 +1,75 @@
+import { useState } from "react";
+import Slider from "../Slider/Slider";
+import NavButton from "../NavButton/NavButton";
+import Guide from "../Guide/Guide";
+import * as Styled from "./ToolMoodMeasurer.styled";
+
+export default function ToolMoodMeasurer({ page, setPage }) {
+  const [before, setBefore] = useState(0.5);
+  const [after, setAfter] = useState(0.5);
```

Comment:
**Just an egghead 🥚 suggestion**
_You can use derived state in your component to simplify the state management._

_Not tested example_

```js
const [mood, setMood] = useState({ before: 0.5, after: 0.5 });

function handleSliderChange(event) {
  setMood((prevMood) => ({
    ...prevMood,
    [page === 0 ? 'before' : 'after']: event.target.value,
  }));
}
```

Reply:
thank you for the idea but ... no :D (can't wrap my 🥚-head around it...)